simapp/go.mod

```diff
@@ -218,6 +218,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
+replace cosmossdk.io/client/v2 => ../client/v2
```

Comment:
in a follow-up this should be removed and a pseudo version of client/v2 should be used.